esp/src/src-react/components/WorkunitSummary.tsx

```diff
@@ -196,6 +213,7 @@ export const WorkunitSummary: React.FunctionComponent<WorkunitSummaryProps> = ({
                             "owner": { label: nlsHPCC.Owner, type: "string", value: workunit?.Owner, readonly: true },
                             "jobname": { label: nlsHPCC.JobName, type: "string", value: jobname },
                             "description": { label: nlsHPCC.Description, type: "string", value: description },
+                            "potentialSavings": { label: nlsHPCC.PotentialSavings, type: "string", value: `${formatCost(potentialSavings)} (${Math.round((potentialSavings / totalCosts) * 10000) / 100}%)`, readonly: true },
```

Comment:
What's the 10000 factor?

Reply:
Just a trick to ensure two decimal places.